core/src/main/scala/org/apache/spark/util/HadoopFSUtils.scala

```diff
@@ -69,6 +69,38 @@ private[spark] object HadoopFSUtils extends Logging {
       ignoreMissingFiles, ignoreLocality, parallelismThreshold, parallelismMax)
   }
 
+  /**
+   * Lists a collection of paths recursively with a single API invocation.
+   * Like parallelListLeafFiles, this ignores FileNotFoundException on the given root path.
+   *
+   * This is able to be be called on both driver and executors.
```

Comment:
```suggestion
   * This is able to be called on both driver and executors.
```

Reply:
Thank you. I'll update with the above typo together.